src/Microsoft.Tye.Hosting/wwwroot/js/logview.js

```diff
@@ -0,0 +1,6 @@
+/** Scroll to the bottom of the input element */
+function logviewScrollToBottom(elementId) { 
```

Comment:
It looks like my ad-blocker and or browser is blocking this script by default.

Reply:
![image](https://user-images.githubusercontent.com/8302101/113640470-598de480-9630-11eb-8f0a-9ac38ede3f1e.png)

Reply:
Interesting.. perhaps because of the name?

Reply:
@jkotalik can you reproduce if you change the name to 'autoscroll.js'?

Reply:
Privacy filter/blocker contains /logview.js as a filter https://easylist-downloads.adblockplus.org/easyprivacy.txt ;)

Reply:
Good find, I renamed it to 'utilities.js'